### ISSUE TITLE
Nether/End won't generate

### ISSUE THREAD

**Issue by untuned** [BlueMap-Minecraft/BlueMap#27]
Just shows 0 tiles.

![](https://untnd.me/HopBEq9Lt)

**Comment by TBlueF** [comment]
@untuned Did you just create the nether/end? The tiles might not have been saved to disk yet. Try `/save-all` before `/bluemap render` please :)

**Comment by untuned** [comment]
No, these worlds have existed since the start of the server.  Even if I stand in the nether, it doesn't do anything. Console shows no errors either.

![](https://untnd.me/d03L0VqwT)

**Comment by TBlueF** [comment]
@untuned hmm.. in bluemaps configuration for the nether map, is `world:` pointing to the correct dimension folder?
*(If you are unsure, maybe you could show me your config and the folder-structure of your worlds)*

**Comment by untuned** [comment]
It's definitely correct, the world is named "world_nether".
```
# Here another example for the Nether-Map
	{
		id: "nether"
		name: "Nether"
		world: "world_nether"
		
		skyColor: "#290000"
		
		renderCaves: true
		ambientLight: 0.6
		
		# We slice the whole world at y:90 so every block above 90 will be air.
		# This way we don't render the nethers ceiling.
		maxY: 90
		renderEdges: true
	}
```

**Comment by TBlueF** [comment]
@untuned ooh .. okay thats interesting .. i am not that familliar with bukkit .. i just tried it myself with a server and noticed that bukkit is doing sth really weird with it's dimensions:
You actually have to configure `world_nether/DIM-1` and `world_the_end/DIM1` for it to work .. so the default values i put in the config are wrong ^^'
Try that and it should work, tell me if it did! Thanks for the report! :)

**Comment by untuned** [comment]
> @untuned ooh .. okay thats interesting .. i am not that familliar with bukkit .. i just tried it myself with a server and noticed that bukkit is doing sth really weird with it's dimensions:
> You actually have to configure `world_nether/DIM-1` and `world_the_end/DIM1` for it to work .. so the default values i put in the config are wrong ^^'
> Try that and it should work, tell me if it did! Thanks for the report! :)

Trying that makes it so I can't render it apparently.
![](https://untnd.me/es8Nk3wcL)
![](https://untnd.me/OkX6xa_Su)

**Comment by TBlueF** [comment]
Oof, okay i think know why. That's a bug in the code then.. I'll fix that.

**Comment by TBlueF** [comment]
Fixed with 015b38311d11420f79c6afc3843b679f406fa25d
